cms/src/main/resources/api-specification/codegen-templates/index.mustache

```diff
@@ -70,7 +70,9 @@
                                             <p></p>
                                             <br/>
 
-                                            <h2>Request</h2>
+                                            {{#isAnyTrue Boolean hasPathParams hasHeaderParams hasBodyParam hasFormParams hasQueryParams}}
```

Comment:
I'm afraid that the use of `Boolean` here is incorrect; admittedly, it probably doesn't break anything but it is misleading. 

If I remember what Handlebars documentation says, the first argument either represents the current context or even is optional.

It would be best to remove this argument if it's indeed optional, or replace it with something more meaningful.

Reply:
Fixed